Rakefile

```diff
@@ -0,0 +1,100 @@
+require 'json'
+require 'fileutils'
+require 'mkmf'
```

Comment:
Are we using `mkmf` anywhere? I'm not seeing any references, but it might be pulling it helper stuff.